packages/wrangler/src/containers/config.ts

```diff
@@ -46,6 +46,81 @@ function convertContainerAffinitiesForApi(
  * We set defaults here too, because we can assume that if the value is undefined,
  * we want to revert to the default rather than inheriting from the prev deployment
  */
+
+
+/**
+ * Auto-generate Durable Object bindings and migrations for containers
+ * This reduces wrangler.toml boilerplate by creating necessary DO config automatically
+ */
+export const autoGenerateContainerDurableObjects = (config: Config): void => {
+	if (!config.containers || config.containers.length === 0) {
+		return;
+	}
+
+	// Initialize durable_objects structure if it doesn't exist
+	if (!config.durable_objects) {
+		config.durable_objects = { bindings: [] };
+	}
+	if (!config.durable_objects.bindings) {
+		config.durable_objects.bindings = [];
+	}
+
+	// Initialize migrations if it doesn't exist
+	if (!config.migrations) {
+		config.migrations = [];
+	}
+
+	for (const container of config.containers) {
+		// Auto-generate class_name if not provided
+		if (!container.class_name) {
+			// Convert container name to PascalCase for class name
+			container.class_name = container.name
+				.split(/[-_]/)
+				.map(word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase())
+				.join('') + 'Container';
+		}
```

Comment:
IIUC this is a naming convention that users have to respect. If they name their class anything else than `nameContainer`, it won't work.

If that is the case, could you please document that convetion in the JS doc for this function and the README.